tests/h/indexer/reindexer_test.py

```diff
@@ -123,8 +123,8 @@ def batchindexer(self, BatchIndexer):
 
     @pytest.fixture
     def es(self):
-        mock_es = mock.Mock(spec=client.Client('localhost', 'hypothesis'))
-        mock_es.index = 'hypothesis'
+        mock_es = mock.create_autospec(client.Client, instance=True,
+                                       spec_set=True, index="hypothesis")
```

Comment:
Given that `index` is now a property...is that keyword arg still necessary in the autospec creation here? Or is this explicitly setting the value of that property?

Reply:
It's setting the value, otherwise `mock_es.index` would be a mock object not a string